src/Core/Children.js

```diff
@@ -17,7 +17,7 @@ var Children = {
 			array = children
 		else if (typeof children.next === 'function')
 			each(children, function (value) {
-				array.push(value)
+				return array.push(value)
```

Comment:
? did you mean `}) return array`

Reply:
In this case it intends to return the index that `.push` returns.